<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsModulesPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsModulesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="LoadModulesLabel">
     <property name="text">
      <string>Don't load modules:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="LoadModulesCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="LoadCommandLineModuleLabel">
     <property name="text">
      <string>Don't load command-line plugins:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="LoadCommandLineModulesCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ExtensionInstallDirectoryLabel">
     <property name="text">
      <string>Extension install path:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkDirectoryButton" name="ExtensionInstallDirectoryButton"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="TemporaryDirectoryLabel">
     <property name="text">
      <string>Temporary directory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkDirectoryButton" name="TemporaryDirectoryButton"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
